templates/emails/github.signup.hbs

```diff
@@ -8,13 +8,24 @@ Subject: Your Open Collective is on its way...
 <p>Hi!</p>
 {{/if}}
 
-<p> Thanks for signing up your GitHub repository: {{group.name}}. We are reviewing new requests as fast as we can. </p>
+<p> Thanks for signing up on Open Collective! Your collective is now live: <a href="{{config.host.website}}/{group.slug}">{{group.name}}</a>. While you wait for approval to start receiving donations, here's a quick guide to make the most out of it: </p>
```

Comment:
I wouldn't say "live" because that implies that it's all set and published and ready.
I'd say. `Thanks ... Your collective has been created and is available at the following URL: `.
...
`Here is a quick guide to customize your collective and make the most out of it`